python/cudf/cudf/core/join/_join_helpers.py

```diff
@@ -97,6 +97,14 @@ def _match_join_keys(
     if pd.api.types.is_dtype_equal(ltype, rtype):
         return lcol, rcol
 
+    if isinstance(ltype, cudf.Decimal64Dtype) or isinstance(
```

Comment:
It is better to be kept under `elif`.

Reply:
I don't think this matters since the above `if` statement returns if it evaluates to `True`.

Reply:
I agree, I just meant in the sense of readability.